_faq.Rmd

```diff
@@ -302,45 +309,54 @@ So most ISO 639-3 codes have only 1 inventory (the long, bottom line).
+    arrange(desc(`Number of inventories`)) %>%
+    filter(`Number of inventories` > 5) %>%
     kable()
 ```
 
 
 # Why are multiple inventories sometimes linked to the same document?
 
-Occasionally, a single document may provide information about multiple phonological inventories. For example, a dissertation that describes and compares three related dialects or languages spoken in a particular region. In that case, three phonological inventories in PHOIBLE corresponding to three doculects might all be linked to the same document, but each inventory is still linked to only one document (it just happens to be *the same document* for those three inventories). One example is [@Terrill1998], a grammar of [Biri](https://en.wikipedia.org/wiki/Biri_language), which contains a chapter describing its dialects, [many of which appear in PHOIBLE](https://phoible.org/languages/biri1256).
+Occasionally, a single document may provide information about multiple phonological inventories. For example, a dissertation that describes and compares three related dialects or languages spoken in a particular region. In that case, three phonological inventories in PHOIBLE corresponding to three doculects might all be linked to the same document, but each inventory is still linked to only one document (it just happens to be *the same document* for those three inventories). One example is @Terrill1998, a grammar of [Biri](https://en.wikipedia.org/wiki/Biri_language), which contains a chapter describing its dialects, [many of which appear in PHOIBLE](https://phoible.org/languages/biri1256).
 
 
 # What are the different "sources" in PHOIBLE?
 
-PHOIBLE contains inventories from various different [contriubutors](https://phoible.org/contributors). These contributions are grouped into so-called "sources", denoted by abbreviation, including (in order of bibliographic):
+PHOIBLE contains inventories from various different
+[contributors](https://phoible.org/contributors). These contributions are
+grouped into so-called "sources", denoted by abbreviations. Here they
+are in the chronological order that they were added to PHOIBLE:
 
 * [SPA](https://github.com/phoible/dev/tree/master/raw-data/SPA): The Stanford Phonology Archive [@CrothersEtAl1979]
-* [UPSID](https://github.com/phoible/dev/tree/master/raw-data/UPSID): The [UCLA Phonological Segment Inventory Database](http://web.phonetik.uni-frankfurt.de/upsid.html) [@Maddieson1984, @MaddiesonPrecoda1990]
-* [AA](https://github.com/phoible/dev/tree/master/raw-data/AA): [Alphabets of Africa](http://sumale.vjf.cnrs.fr/phono/index.htm) [@hartell1993, @chanard2006]
+* [UPSID](https://github.com/phoible/dev/tree/master/raw-data/UPSID): The [UCLA Phonological Segment Inventory Database](http://web.phonetik.uni-frankfurt.de/upsid.html) [@Maddieson1984;@MaddiesonPrecoda1990]
+* [AA](https://github.com/phoible/dev/tree/master/raw-data/AA): [Alphabets of Africa](http://sumale.vjf.cnrs.fr/phono/index.htm) [@hartell1993;@chanard2006]
```

Comment:
must use semicolons to separate multiple references between brackets.

Reply:
Noted!